Core/FileSystems/BlockDevices.cpp

```diff
@@ -62,6 +63,13 @@ u32 BlockDevice::CalculateCRC() {
 	return crc;
 }
 
+void BlockDevice::NotifyReadError() {
+	I18NCategory *err = GetI18NCategory("Error");
+	if (!reportedError_) {
+		host->NotifyUserMessage(err->T("Game disc read error - check crc"), 6.0f);
```

Comment:
"ISO corrupt" instead of "check crc"? Not sure how many users understand CRCs... and if we're gonna keep it, CRC needs to be capitalized.

Reply:
Fair enough - changed.  I initially didn't use ISO in case someone would be pedantic about CSOs, but it doesn't really matter.

-[Unknown]